src/network/esp/include/NetworkManager.h

```diff
@@ -0,0 +1,32 @@
+#ifndef HIVE_CONNECT_NETWORKMANAGER_H
+#define HIVE_CONNECT_NETWORKMANAGER_H
+
+#include "logger/ILogger.h"
+#include <BaseTask.h>
+#include <Task.h>
+
+#include "esp_wifi.h"
+
+class NetworkManager {
```

Comment:
Will you have an interface that will be used for the ROS build too?  I guess you doc will be on the interface, if not maybe add some doc to the class

Reply:
Addded doc to the class. There is no common interface now for ROS and ESP, because the manager is too specific for ROS, and all of its calls are handled by the bsp.

Reply:
In the future, there will be a common section, just not necessarily for the manager itself